CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Fixed flyout component resize handle to not have extra styles in Firefox. [#15](https://github.com/blackbaud/skyux-flyout/pull/15)
+- Fixed flyout component component to only listen to `mousemove` and `mouseup` events while being resized. [#16](https://github.com/blackbaud/skyux-flyout/pull/16)
```

Comment:
Is there a way to describe this bug fix without mentioning the internal workings? What was the overarching issue being solved?

Reply:
Change detection/listeners were firing on every mouse move even when they shouldn't. I just copied what we have in `grids` changelog for this same change. Open to any other ideas. Maybe @blackbaud-johnly has some guidance when he looks at it?

Reply:
Gotcha. I was thinking more along the lines of "performance enhancements". Either way is fine with me.

Reply:
I'd tighten this up to "Fixed flyout component to only listen to `mousemove` and `mouseup` events when resizing."

Steve, if we want to obfuscate the internal workings, I'd suggest "Updated flyout component to improve performance." I defer to you on which version is more useful, but if we go with the second version, I think we should also update the entry in the grids changelog.

Reply:
I went ahead with "Updated flyout component to improve performance." 

I'll make a note to update grids later today